django-backend/fecfiler/transactions/migrations/0007_transaction__calendar_ytd_per_election_office_and_more.py

```diff
@@ -0,0 +1,172 @@
+# Generated by Django 4.2.11 on 2024-05-21 20:49
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("transactions", "0006_independent_expenditure_memos_no_aggregation_group"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="transaction",
+            name="_calendar_ytd_per_election_office",
+            field=models.DecimalField(
+                blank=True, decimal_places=2, max_digits=11, null=True
+            ),
+        ),
+        migrations.AddField(
+            model_name="transaction",
+            name="aggregate",
+            field=models.DecimalField(
+                blank=True, decimal_places=2, max_digits=11, null=True
+            ),
+        ),
+        migrations.AddField(
+            model_name="transaction",
+            name="loan_payment_to_date",
+            field=models.DecimalField(
+                blank=True, decimal_places=2, max_digits=11, null=True
+            ),
+        ),
+        migrations.RunSQL(
```

Comment:
Just an idea - I wonder if it would be worthwhile and possible to break this up into several smaller functions (similar to calculations.py) for each of the schedules.